build/media-suite_compile.sh

```diff
@@ -330,6 +330,35 @@ if [[ $curl = y ]]; then
     enabled mbedtls && curl=mbedtls
     [[ $curl = y ]] && curl=schannel
 fi
+
+_check=(libunistring.{,l}a)
+if [[ $mediainfo = y || $bmx = y || $curl != n ]] && ! files_exist "${_check[@]}" &&
```

Comment:
If the checks are all the same, just place it in a single if.

Reply:
Also, are all these three libraries not available statically from msys2?

Reply:
There's no dependency tracking. libunistring is needed by what? curl? libidn2 is required by what? libpsl is required for curl, I assume, but this is not being made explicit anywhere.

I see these three libs do exist in msys. So what's the issue with using them there? If it's the static libs being broken, can they not be fixed in msys? I'd rather not fill the suite with additional maintenance burdens unnecessarily.

Reply:
curl is being built specifically and optionally by the user because it's an "end" executable/lib with selectable features. There's no option to not include libidn/libpsl. I don't personally care for them, for example. Why am I wasting time compiling things I don't care for?
I'm not saying to add a .ini option to not include these, but they should at least be disableable through checking curl configuration options, so the user can explicitely add a --without-{libidn2,libpsl} to not waste time on these.